joblib/testing.py

```diff
@@ -44,3 +49,37 @@ def assert_raises_regex(expected_exception, expected_regexp,
             if not_raised:
                 raise AssertionError("Should have raised %r" %
                                      expected_exception(expected_regexp))
+
+
+def check_subprocess_call(cmd, timeout=1, stdout_regex=None):
+    """Runs a command in a subprocess with timeout in seconds.
+
+    Also checks returncode is zero and stdout if stdout_regex is set.
+    """
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE)
+
+    def kill_process():
+        proc.kill()
```

Comment:
Forget about this comment. It's done line 65. It's strange that coveralls doesn't detect it.